### ISSUE TITLE
False positive for parentheses around condition of if

### ISSUE THREAD

**Issue by dominikh** [rubocop/rubocop#270]
The following snippet will complain about parentheses around the condition, even though they only surround part of it, and correctly so.

``` ruby
if (a + b).c()
end
```
